### PR TITLE
dotCMS/core#18019 fix Update dot-binary-file web component to support maxFileLength attr

### PR DESCRIPTION
**Pull request by alfredo-dotcms**
### Proposed Changes
In the dot-binary-file web component we have the property `maxFileLength` but we don't do anything with it.

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

File: libs/dotcms-webcomponents/src/components/contenttypes-fields/dot-binary-file/dot-binary-file.tsx [hunk 1]
```diff
@@ -297,7 +307,6 @@ export class DotBinaryFileComponent {
                 </dot-label>
                 {getTagHint(this.hint)}
                 {getTagError(this.shouldShowErrorMessage(), this.getErrorMessage())}
-                <dot-error-message>{this.errorMessage}</dot-error-message>
```
Comment (alfredo-dotcms) [inline libs/dotcms-webcomponents/src/components/contenttypes-fields/dot-binary-file/dot-binary-file.tsx:300]: not used

File: libs/dotcms-webcomponents/src/components.d.ts [hunk 2]
```diff
@@ -1591,6 +1603,10 @@ declare namespace LocalJSX {
           * (optional) Disables field's interaction
          */
         "disabled"?: boolean;
+        /**
```
Comment (hmoreras) [inline libs/dotcms-webcomponents/src/components.d.ts:1606]: duplicate attr ?!

**Review by hmoreras — CHANGES_REQUESTED** [review]

**Review by hmoreras — DISMISSED** [review]

File: libs/dotcms-webcomponents/src/components/contenttypes-fields/dot-binary-file/components/dot-binary-upload-button/dot-binary-upload-button.tsx [hunk 3]
```diff
@@ -32,6 +32,10 @@ export class DotBinaryUploadButtonComponent {
     @Prop({ reflect: true })
     disabled = false;
 
+    /** (optional) Set the max file size limit  */
+    @Prop({ reflect: true, mutable: true })
```
Comment (fmontes) [inline libs/dotcms-webcomponents/src/components/contenttypes-fields/dot-binary-file/components/dot-binary-upload-button/dot-binary-upload-button.tsx:36]: Why mutable?

**Review by fmontes — CHANGES_REQUESTED** [review]

File: libs/dotcms-webcomponents/src/components/contenttypes-fields/dot-binary-file/components/dot-binary-upload-button/dot-binary-upload-button.tsx (hunk 3 above)
Comment (alfredo-dotcms) [reply libs/dotcms-webcomponents/src/components/contenttypes-fields/dot-binary-file/components/dot-binary-upload-button/dot-binary-upload-button.tsx:36]: no reason

**Review by hmoreras — APPROVED** [review]

**Review by fmontes — APPROVED** [review]

**Pull request merged**